### DISCUSSION TITLE
Swift Binary Packages

### DISCUSSION THREAD

**Discussion by capjan** [krzyzanowskim/CryptoSwift#974] [General]
Sorry for using this issue as discussion.
I read about the performance problems for the swift package deployment and debug builds.

What about an additional deployment as binary swift package?

**Comment by krzyzanowskim** [comment]
oh, it's complicated. Been there, and reverted https://github.com/krzyzanowskim/CryptoSwift/releases/tag/1.3.8 however the 1.4.0 release page has xcframework available to download https://github.com/krzyzanowskim/CryptoSwift/releases/tag/1.4.0 I guess you can just get it and use it on your side.

For the SPM distribution, that'd need a separate repository to work, that is a bit of ha hussle

**Comment by capjan** [comment]
Thanks for the reply and comments.

**Comment by krzyzanowskim** [comment]
I'm gonna close it now. I'll review binary distribution after WWDC21 and next version of Xcode
